### PR TITLE
data:shreyanshs31

### PR DESCRIPTION
**Pull request by shreyanshs31**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7740"></a>

### CODE REVIEW

File: data/shreyanshs31.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Shreyansh Shukla",
+  "displayStatsPublic":"true",
+  "bio": "Student👨🏽‍🎓 • DSA👨🏽‍💻 • Python • Java •  Learning in Public👤 • Love Open Source💜"
```
Comment (Krishnapro) [inline data/shreyanshs31.json:4]: Add `,` at the end of line 4

**Review by Krishnapro — CHANGES_REQUESTED** [review]
The rest of the code looks good to me :+1:

File: data/shreyanshs31.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  "type": "personal",
+  "avatar":"https://github.com/shreyanshs31.png",
```
Comment (RohanSasne) [inline data/shreyanshs31.json:6]: No need to add an avatar i guess, It automatically fetches profile picture which you use for github
```suggestion
```

**Review by RohanSasne — CHANGES_REQUESTED** [review]

File: data/shreyanshs31.json (hunk 1 above)
Comment (SaraJaoude) [reply data/shreyanshs31.json:4]:
```suggestion
  "bio": "Student👨🏽‍🎓 • DSA👨🏽‍💻 • Python • Java •  Learning in Public👤 • Love Open Source💜",
```

File: data/shreyanshs31.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Shreyansh Shukla",
+  "displayStatsPublic":"true",
+  "bio": "Student👨🏽‍🎓 • DSA👨🏽‍💻 • Python • Java •  Learning in Public👤 • Love Open Source💜",
+  "type": "personal",
```
Comment (SaraJaoude) [inline data/shreyanshs31.json:5]: the following are no longer necessary:
```suggestion
  "bio": "Student👨🏽‍🎓 • DSA👨🏽‍💻 • Python • Java •  Learning in Public👤 • Love Open Source💜",
```

**Review by SaraJaoude — APPROVED** [review]
I have committed the changes suggested by the other reviewers and made some further changes for the JSON to be valid - I will now merge this PR.

**Pull request merged**